### ISSUE TITLE
Default npm -g install dependency conflicts with leveldown v1.2

### ISSUE THREAD

**Issue by bwarminski** [mhart/kinesalite#13]
Noticed this issue when building a fresh Docker image:

Repro steps:
1. On fresh machine without leveldown or levelup in lib/node_modules: npm install -g kinesalite
2. ~# kinesalite

Stacktrace is:

```
/usr/lib/node_modules/kinesalite/node_modules/levelup/lib/util.js:42
    throw new LevelUPError(
          ^
LevelUPError: Installed version of LevelDOWN (1.2.2) does not match required version (~1.1.0)
    at getLevelDOWN (/usr/lib/node_modules/kinesalite/node_modules/levelup/lib/util.js:42:11)
    at LevelUP.open (/usr/lib/node_modules/kinesalite/node_modules/levelup/lib/levelup.js:114:37)
    at new LevelUP (/usr/lib/node_modules/kinesalite/node_modules/levelup/lib/levelup.js:87:8)
    at LevelUP (/usr/lib/node_modules/kinesalite/node_modules/levelup/lib/levelup.js:47:12)
    at Object.create (/usr/lib/node_modules/kinesalite/db/index.js:31:12)
    at kinesalite (/usr/lib/node_modules/kinesalite/index.js:24:26)
    at Object.<anonymous> (/usr/lib/node_modules/kinesalite/cli.js:26:35)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

Changing the leveldown dependency from "^1.0.6" to "1.1" fixes the runtime error:

```
sed -i "s/\"leveldown\": \"\^1.0.6\"/\"leveldown\": \"1.1\"/g" /git/kinesalite/package.json
```

**Comment by evansolomon** [comment]
Fwiw, I don't see this error on a fresh docker container.

```
$ docker run --rm -ti node bash
$ npm i -g kinesalite
$ kinesalite
Listening at http://:::4567
```

**Comment by mhart** [comment]
Ugh, that's due to levelup and its strict version checking:

https://github.com/Level/levelup/blob/master/lib/util.js#L41

It uses the semver string specified in its package.json here:

https://github.com/Level/levelup/blob/master/package.json#L50

Which unfortunately isn't the npm default anymore (`^` vs `~`)

I didn't realise that it did this until now – thanks for the issue.

@evansolomon you'll only see it if you pass a path in, otherwise it doesn't use leveldown at all

**Comment by mhart** [comment]
Fixed in v1.6.1

**Comment by mhart** [comment]
Just commented over at https://github.com/Level/levelup/issues/351 to see if there's some guidance on how to handle this going forward.
